docs/install/Knative-custom-install.md

```diff
@@ -283,6 +283,15 @@ commands below.
    kubectl delete deploy knative-ingressgateway -n istio-system
    ```
 
+   You will also need to delete the following resource:
+   ```
+   kubectl delete statefulset/controller-manager -n knative-sources
+   ```
+   While the deletion of this resource during the upgrade process will
```

Comment:
Does this paragraph (lines 290-294) mean that deleting this resource won't break current eventing implementations in the time between deleting this resources and upgrading to the next version? If so, let's just state that.

If not, can you try to simplify + clarify this? I'm not sure I understand.

Reply:
Well, it won't break anything but it will put any changes made "on hold". Meaning, your `kuebctl apply <some-eventing-sources-resource>` will be accepted but not actually take effect until the rest of the install finishes. That's what I was trying to say...